app/helpers/view_helper.rb

```diff
@@ -49,6 +49,17 @@ def govuk_button_link_to(body, url, html_options = {}, &_block)
     link_to(body, url, html_options)
   end
 
+  def govuk_button_to(name, options = {}, html_options = {})
+    html_options = {
+      class: prepend_css_class('govuk-button', html_options[:class]),
+      role: 'button',
+      data: { module: 'govuk-button' },
+      draggable: false,
+    }.merge(html_options)
+
+    button_to(name, options, html_options)
+  end
+
```

Comment:
👍🏼 I hadn't seen this helper before. It seems like it could do with some refactoring so that govuk components are moved into their own helper. I'll add a card on the tech debt boart.